go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/timescale/promscale
 
-go 1.17
+go 1.18
```

Comment:
Not really, I think we are using 1.19 for building the image, this is just the minimum version required. I had to bump it because pgx uses generics. Do you want me to bump it to 1.19?

Reply:
If it's no effort I'd rather pick 1.19 :)